src/plugins/vis_default_editor/public/_sidebar.scss

```diff
@@ -4,6 +4,11 @@
 
 .visEditorSidebar {
   min-width: $vis-editor-sidebar-min-width;
+
+  // a hack for IE, issue: https://github.com/elastic/kibana/issues/66586
+  > .visEditorSidebar__formWrapper {
+    flex-basis: auto;
```

Comment:
I don't think this should be wrapped with `internetExplorerOnly` mixin.
Checked in all browsers except Safari, works fine.
Actually, this just overwrites EUI rule, which seems was created for another IE bug 😄 :

![image](https://user-images.githubusercontent.com/31325372/82045496-d9792c00-96b7-11ea-98c7-11d9586457f0.png)